lxd/cluster/recover.go

```diff
@@ -134,8 +134,12 @@ func Reconfigure(database *db.Node, raftNodes []db.RaftNode) error {
 
 		return err
 	})
-	if err != nil || info == nil {
-		return errors.Wrap(err, "Failed to determine node role")
+	if err != nil {
+		return fmt.Errorf("Failed to determine node role: %w", err)
+	}
+
+	if info == nil {
+		return fmt.Errorf("This node has no raft role")
```

Comment:
We say "member" not "node" in user facing errors/logs.

Reply:
Switched to "cluster member".

Reply:
thanks!